spec/truffle/java_exception_spec.rb

```diff
@@ -21,7 +21,7 @@
   it "includes the first lines of the Java stacktrace for uncaught Java exceptions" do
     lambda { Truffle::Debug.throw_java_exception 'message' }.should raise_error { |e|
       message = e.message.gsub(/:\d+/, ':LINE')
-      message.lines[2].should == "\torg.truffleruby.debug.TruffleDebugNodesFactory$ThrowJavaExceptionNodeFactory$ThrowJavaExceptionNodeGen.execute(TruffleDebugNodesFactory.java:LINE)\n"
+      message.lines[2].start_with?("\torg.truffleruby.debug.TruffleDebugNodesFactory$ThrowJavaExceptionNodeFactory$ThrowJavaExceptionNodeGen.execute").should be_true
```

Comment:
How about:
```ruby
message.lines[2].should =~ /^\torg\.truffleruby\.debug\.TruffleDebugNodesFactory$ThrowJavaExceptionNodeFactory$ThrowJavaExceptionNodeGen\.\w+\(TruffleDebugNodesFactory\.java:LINE\)$/
```
Not pretty, but it's more precise and make it easier to have changing parts.
BTW, I'd like to add the start_with/end_with matchers to mspec, but not sure I'll find the time.